dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -60,7 +71,7 @@ startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
-dependent_startup_wait_for=zebra:running
+dependent_startup_wait_for=zsocket:exited
```

Comment:
Please change it to "staticd:running". This will make sure that bgp daemon starts after static daemon. In one case, static route had to be sent to BGP peer. Static route is config dependent and should come up before bgp.

Reply:
does this matter? how does static route send route to bgp peer? i assume it is via zebra, so staticd connect to zebra and send route to zebra, later as soon as bgp connects, zebra will send those routes to bgpd. why bgpd has dependency on staticd?

Reply:
@shi-su , can you please test this scenario. Create a 2 node topology and "redistribute static" in BGP. have a static route, save the FRR config and reboot. we should see the static route in the BGP neighbor.

Reply:
hi @sudhanshukumar22 , we are not familiar with that scenario, can you help to test?

Reply:
@shi-su , please checkin your changes. Once we take the changes, if there are issues, I will file another PR.